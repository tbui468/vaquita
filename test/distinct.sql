if exists drop database sol;
create database sol;
open sol;

create table planets (
                        id int key,
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        id,
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        (1, "Venus", 2.32, 0, 92.0, true),
        (2, "Venus", null, 1, 1.0, false),
        (3, "Mars", 2.32, 5, 934.3, true),
        (4, "Mars", 4.23, 2, 0.01, true);

select distinct name, ring_system from planets;
close sol;
drop database sol;
exit;
