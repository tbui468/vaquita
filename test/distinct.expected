created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
name, ring_system
Venus, true
Venus, false
Mars, true
closed database sol
dropped database sol
